guide/index.md

```diff
@@ -135,7 +127,10 @@ const styles = StyleSheet.create({
 Here is how it all could work together:
 
 :::info
-We are using expo-camera to request camera permissions. You can install and build it using the following command:
+
+We are using expo-camera to request camera permissions. You can install and build it using the following
+
+command:
```

Comment:
**question:** Why new line here?

Reply:
Prettier changes 
```
:::info
We are
```
to 
```
:::info We are
```

So extra line is required to prevent it (I think that markdown formatting is not fully stable 🤔 )